src/app/mail/mail-sidebar/compose-mail/compose-mail.component.html

```diff
@@ -106,7 +106,7 @@
         <textarea class="mail-compose-control" name="subject" [(ngModel)]="mailData.subject"
                   [disabled]="action===mailAction.REPLY || action===mailAction.REPLY_ALL"
                   (ngModelChange)="valueChanged$.next(mailData.subject)"
```

Comment:
Why don't you add it in `ngModelChange` instead of `blue` like below?
(ngModelChange)="valueChanged$.next(mailData.subject); onSubjectChange(mailData.subject)"

Reply:
Okay, let me update. :)